lamindb/_can_curate.py

```diff
@@ -144,7 +154,9 @@ def _inspect(
     using_key = queryset.db
     if isinstance(source, Record):
         _check_source_db(source, using_key)
-        queryset = queryset.filter(source=source).all()
+        # if not strict mode, do not filter by source here because we want to inspect all records in the DB first
```

Comment:
Can the comment be more comprehensive, please?

"if in strict mode, restrict the query to one particular ontology source"

"otherwise, search across records from all ontology sources present in the registry"